dali/pipeline/operator/op_schema.h

```diff
@@ -656,12 +656,20 @@ used with DALIDataType, to avoid confusion with `AddOptionalArg<type>(name, doc,
    * own the storage and the associated input should be included in double-buffering
    * whenever the output should.
    *
+   * TODO(klecki): Introduce additional class of Pass Through that enables partial mappings
+   * (splitting one input into multiple outputs or merging several inputs to one output).
+   *
    * @param inout - tells which inputs are passed through to which outputs.
-   *                Multiple inputs can be passed through to one output (at
-   *                least potentially, e.g. when conditionally forwarding
-   *                one of inputs to the output), but not vice versa.
+   *                Only bijective mappings are allowed.
    */
   DLL_PUBLIC inline OpSchema &PassThrough(const std::map<int, int> &inout) {
```

Comment:
If it's mapping [1, n) -> [1, n), wouldn't vector of size n just be enough for representing it? Just a note. If it's not that or if you think map is better, ignore this comment.

Reply:
I left the possibility for a partial mapping, maybe I should not call it a bijective mapping (partial bijective functions exist).